index.js

```diff
@@ -1106,12 +1114,15 @@
   //. Function wrapper for [`fantasy-land/ap`][].
   //.
   //. `fantasy-land/ap` implementations are provided for the following
-  //. built-in types: Array and Function.
+  //. built-in types: Array, Object, and Function.
   //.
   //. ```javascript
   //. > ap([Math.sqrt, x => x * x], [1, 4, 9, 16, 25])
   //. [1, 2, 3, 4, 5, 1, 16, 81, 256, 625]
   //.
+  //. > ap({x: Math.sqrt, y: x => x * x}, {x: 16, y: 10, z: 1})
+  //. {x: 4, y: 100}
+  //.
```

Comment:
Let's name the properties `a`, `b`, and `c` to avoid the slight confusion arising from `x` being used as both a property name and a parameter name.